src/Stryker.Core/Stryker.Core/InjectedHelpers/MutantControl.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Stryker
+{
+    using Core.InjectedHelpers.Coverage;
+
+    public static class MutantControl
+    {
+        private static HashSet<int> _coveredMutants;
+        private static bool usePipe;
+        private static string pipeName;
+        private static string envName;
+        private static bool captureCoverage;
+        private static CommunicationChannel channel;
+
+        public const string EnvironmentPipeName = "Coverage";
+        
+        static MutantControl()
+        {
+            InitCoverage();
+            if (usePipe)
+            {
+                AppDomain.CurrentDomain.ProcessExit += CurrentDomain_ProcessExit;
+            }
+        }
+
+        public static void InitCoverage()
+        {
+            ActiveMutation = int.Parse(Environment.GetEnvironmentVariable("ActiveMutation") ?? "-1");
+            if (channel != null)
+            {
+                channel?.Dispose();
+                channel = null;
+            }
+            var coverageMode = Environment.GetEnvironmentVariable(EnvironmentPipeName)??string.Empty;
+            if (coverageMode.StartsWith("pipe:"))
```

Comment:
as of now, the usage of environment variables as a communication channel remains an internal option. I propose to wait for real world feedback before making the option available and/or dropping the alternate strategy.
Architecture for this can be improved. For example, we can use the VsTest configuration file to provide configuration data for the coverage collector, instead of using the environment.

But I think it can wait for a further version, as it does not raise issues or concerns as far as I am aware of.